index.js

```diff
@@ -1,45 +1,17 @@
 'use strict';
 
-const apolloClientDefaultPackages = [
-  'apollo-cache',
-  'apollo-cache-inmemory',
-  'apollo-client',
-  'apollo-link',
-  'apollo-link-context',
-  'apollo-link-http',
-  'graphql',
```

Comment:
graphql-js is the problematic dependency.  https://github.com/graphql/graphql-js/issues/1272